clients/mediaconvert.go

```diff
@@ -481,7 +481,7 @@ func copyFile(ctx context.Context, sourceURL, destOSBaseURL, filename, requestID
 
 	content := io.TeeReader(c, &writtenBytes)
 
-	err = UploadToOSURL(destOSBaseURL, filename, content, 5*time.Minute)
+	err = UploadToOSURL(destOSBaseURL, filename, content, 30*time.Minute)
```

Comment:
We need this one for mp4 files, not only for web3.strorage since mp4 can be very big. In the long run,we should make the timeout dependent on the file size.

Reply:
Will this cause issues with Studio timeouts? 
https://github.com/livepeer/studio/blob/23b1aa453bbef83981ad2547169c7149f0436e35/packages/api/src/store/task-table.ts#L7-L9
If we're stuck in `UploadToOSURL`, is our callback to Studio `ReportProgress` called concurrently? 
cc @victorges

Reply:
That is a very good point! I'll move this change into a separate PR and we can discuss it there (since it's not strictly related to saving data, but more to bigger files => mp4)